modules/ROOT/pages/_partials/common-releasenotes.adoc

```diff
@@ -132,13 +132,13 @@ The version of Inter-Sync{nbsp}Gateway useable depends on the Sync Gateway versi
 ====
 Inter-Sync Gateway Replication versions available for use on an active node::
 
-* Pre-Sync Gateway 2.8 -- only Inter-Sync Gateway Replication version 1 is available
-* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication version 2 and Inter-Sync Gateway Replication version 1 (deprecated)
+* Pre-Sync Gateway 2.8 -- only SG{nbsp}Replicate is available
+* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication and SG{nbsp}Replicate (deprecated)
 
 SG versions compatible as a passive node as part of SGR1/SGR2 replications::
 
-* Inter-Sync Gateway Replication v1 can target any Inter-Sync Gateway Replication version
-* Inter-Sync Gateway Replication v2 can only target Inter-Sync Gateway Replication v2+
+* SG{nbsp}Replicate can target any Inter-Sync Gateway Replication version
```

Comment:
I do not know where this shows up. Whats the context? Seems very confusion the way its worded 
Why not just have a matrix which shows versions of Sync Gateway and what inter-sync gateway replications  options exist ?

Reply:
GENERAL:  Use "Inter-Sync Gateway Replication introduced in Sync Gateway 2.8" to refer to new one ..Its wordy but clearer 
(Treat "SG-Replicate" as  the version of Inter-Sync Gateway Replication protocol  pre SGW 2.8 )

Reply:
@priya Can it just be "inter-Sync Gateway replication (2.8+)" ... or where contextually appropriate have 'introduced at 2.8' as a footnote?